src/components/page-template.jsx

```diff
@@ -15,6 +15,23 @@ const PageTemplate = (props) => {
 
 PageTemplate.propTypes = {
   data: PropTypes.shape({
```

Comment:
Because we are only pulling out the date and the url, I think it will be neater to have this interpretted in PageTemplate function above, just before the return, and just pass through the date and url instead of the entire graphql response. I'm putting this here because I can't comment on the actual line as it wasn't changed in this PR 😆

Reply:
+1